airflow/pipe_events_fishing_dag.py

```diff
@@ -0,0 +1,46 @@
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.operators.bash_operator import BashOperator
+
+from pipe_tools.airflow.models import DagFactory
+
+
+class PipelineDagFactory(DagFactory):
+    def source_date_range(self):
+        # The scored messages only have logistic scores for a couple of days
+        # while we accumulate the amount of data we need to refine that score
+        # with the nnet model. We need to shift the intervals' start date so
+        # that it reprocesses the events for up to 7 days before the current
+        # day.
+        if self.schedule_interval == '@daily':
+            return '{{ ds + macros.dateutil.relativedelta.relativedelta(days=-7) }}', '{{ ds }}'
```

Comment:
@andres-arana as we spoke, the `days=-7` should take the `BUFFER_DAYS` variable instead.